### DISCUSSION TITLE
Function keys as standard turned off but they don't work

### DISCUSSION THREAD

**Discussion by divdax** [MonitorControl/MonitorControl#1458] [Q&A]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I installed this app. It didn't work so i deleted it. After that my setting for the functional keys in mac os doesn't work anymore.

    Turn on "Use F1, F2, etc. keys as standard function keys".

I only can access my media keys with "fn" even when the option in the settings of mac os is disabled.
What i can do to disable my functional keys as default?

**Comment by waydabber** [comment]
Honestly, I am not sure why this is happening as the app does not interfere with this setting at all, not while installed or after it is uninstalled. I'll move it to discussions, maybe somebody experienced something similar and might have an insight.

**Reply by reggoboy** [reply]
I was going to offer the same feedback to developers, and then I saw this thread.

I don't use F keys for (almost?) anything on the Mac, and neither do most users, which is why Apple defaults their use to their special functions: volume, brightness, Mission Control, audio transport, etc. 

So in MonitorControl, I want it to behave the same way that MacBook, iMac, and other Apple-OEM screens behave: without having to press the "fn" modifier key to change brightness.

But when I try to enter those key presses under the Keyboard settings, no values are visibly entered, and no keyboard control is achieved. I can only enter values when I press the "fn" modifier key, which causes "F1" and "F2" to appear there. You can see my discussion about it here:

https://talk.tidbits.com/t/apple-confirms-no-more-27-inch-imacs/25201/86?u=dave1

Can this be fixed? I prefer this product over Lunar, except in this regard.

**Reply by reggoboy** [reply]
No updates here? It’s too bad. This is the only reason I had to settle on Lunar.

**Reply by waydabber** [reply]
Native brightness and volume keys should work just fine. If they do not work, accessibility permissions might not be set up properly. You can try and see if it works with [BetterDisplay](https://betterdisplay.pro). Take a look at Settings (gear icon in the menu)/Keyboard in the app as it will clearly show if there is a permissions issue.
